cmds/uinit/uinit.go

```diff
@@ -37,17 +37,17 @@ var (
 )
 
 func tczSetup() error {
-	g, err := filepath.Glob(tczs)
+	pt, err := filepath.Glob(tczs)
```

Comment:
Why did this change? g kind of means glob, but pt means? I suspect this is the result of an errant global replace.

Reply:
You're correct. All other uses of g were referencing a GPT, which is now a PartitionTable. I'll fix this and we should be good to go.